src/componentes/GridItems/GridItems.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import './GridItems.css';
+import {CoffeeCard} from '../../componentes';
+import GridCards from '../../data/GridCards';
```

Comment:
Create an entry point to data folder index.js 
Variables name should a camel case 
```suggestion
import gridCards from '../../data';
```